server/service.go

```diff
@@ -131,11 +132,12 @@ func (m *BoostService) StartHTTPServer() error {
 		Addr:    m.listenAddr,
 		Handler: m.getRouter(),
 
-		ReadTimeout:       0,
-		ReadHeaderTimeout: 0,
-		WriteTimeout:      0,
-		IdleTimeout:       0,
-		MaxHeaderBytes:    m.maxHeaderBytes,
+		ReadTimeout:       time.Duration(config.ServerReadTimeoutMs) * time.Millisecond,
+		ReadHeaderTimeout: time.Duration(config.ServerReadHeaderTimeoutMs) * time.Millisecond,
+		WriteTimeout:      time.Duration(config.ServerWriteTimeoutMs) * time.Millisecond,
+		IdleTimeout:       time.Duration(config.ServerIdleTimeoutMs) * time.Millisecond,
+
+		MaxHeaderBytes: m.maxHeaderBytes,
```

Comment:
max header bytes can also be a config var

Reply:
+1